src/commands/init.ts

```diff
@@ -96,6 +100,22 @@ export default defineCommand({
       process.exit(1)
     }
 
+    if (ctx.args.nightly) {
+      const nightlyNuxtVersion = 'npm:nuxt-nightly@latest'
```

Comment:
what about fetching the latest version from npm? `latest` is often poorly updated (i.e. not updated) by package managers

also, there are currently two nightly channels - the `3x` and `latest` (4x) ones. would it be worth using `--channel` for this purpose, and maybe also `--nightly` which defaults to the latest `3x` tag?